py2app/recipes/zmq.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+def check(cmd, mf):
+    m = mf.findNode("zmq")
+    if m is None or m.filename is None:
+        return None
+
+    dylibs = os.scandir(os.path.join(m.packagepath[0], '.dylibs'))
+    frameworks = [lib.path for lib in dylibs]
+
+    includes = [
+        "zmq.backend.cython",
+    ]
+
+    return {"frameworks": frameworks, "includes": includes}
```

Comment:
Is this really needed? I had expected that py2app would do the right thing automatically here.

Reply:
I had to do the same thing for `rtree` in a previously accepted PR. For both `rtree` and `zmq` the dylibs are copied over with the package into the zip directory (e.g. python39.zip/rtree/lib), but not copied to the Contents/Resources folder unless I do so explicitly in the recipe. If py2app is supposed to catch these automatically, then there must be a bug. Perhaps it is only catching dylibs for packages when they are included in the "packages" list?

However, when packages are included in the packages list, the dylibs reside in two places: in the package and in the Resources directory. Would it be more efficient to add package dylib directories to the RESOURCEPATH environment variable instead? Can zip directories be added to RESOURCEPATH?

Reply:
Or dylibs copied to the Frameworks directory, but excluded when copying the contents of the package (to either collect or Resources/lib/pythonX.X/

Reply:
Sorry, I mis-wrote Resources when I meant Frameworks in the above comments.

Reply:
I'll try to find out what's going on here.  But that shouldn't block merging this PR.